src/views/Types.vue

```diff
@@ -6,8 +6,28 @@
 <script lang="ts">
 
 import { Component, Vue } from 'vue-property-decorator';
+import { ResourceTypes } from '@/apis/rembrandt/Rembrandt';
 
 @Component
-export default class Types extends Vue {}
+export default class Types extends Vue {
+
+  private async mounted() {
+    await ResourceTypes.get();
+    await ResourceTypes.create({
```

Comment:
You still create a test resource type each time the view is called. Maybe we want to remove this before merging in `dev`. (We can also keep it here for further test purposes, but then we should make sure not to about it :smirk:)

Reply:
Since everybody saw that now and understood it, I'll remove it.